src/components/SearchForm/SearchForm.jsx

```diff
@@ -10,22 +10,23 @@ const SearchForm = ({onSerch}) =>
       };
 
     const handleSubmit = (event)=>{
+        event.preventDefault(); 
         console.log(query);
         onSerch(query);
     }
 
     return (
         <>
-        <form className='search-form' >
+        <form className='search-form' 
+                onSubmit={handleSubmit}>
             <input 
                 onChange={handleQueryChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.

To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components